packages/cli/src/commands/generate/script/script.js

```diff
@@ -0,0 +1,74 @@
+import fs from 'fs'
+import path from 'path'
+
+import Listr from 'listr'
+import { paramCase } from 'param-case'
+import terminalLink from 'terminal-link'
+
+import { getPaths, writeFilesTask } from 'src/lib'
+import c from 'src/lib/colors'
+
+const POST_RUN_INSTRUCTIONS = `Next steps...\n\n   ${c.warning(
+  'After writing your script, you can run it with:'
+)}
+
+     yarn rw run <name>
+`
+
+const TEMPLATE_PATH = path.resolve(
+  __dirname,
+  'templates',
+  'script.js.template'
+)
+
+export const files = ({ name }) => {
+  const outputFilename = `${paramCase(name)}.js`
+  const outputPath = path.join(getPaths().api.scripts, outputFilename)
+
+  return {
+    [outputPath]: fs.readFileSync(TEMPLATE_PATH).toString(),
+  }
+}
+
+export const command = 'script <name>'
+export const description = 'Generate a command line script'
+export const builder = (yargs) => {
+  yargs
+    .positional('name', {
+      description: 'A descriptor of what this script does',
+      type: 'string',
+    })
+    .epilogue(
+      `Also see the ${terminalLink(
+        'Redwood CLI Reference',
+        'https://redwoodjs.com/reference/command-line-interface'
+      )}`
+    )
+}
```

Comment:
This part should allow for a positional argument so you can do `yarn rw run <name>` but right this is not working.

I get the following message `Unknown argument: push`

```
$ /Users/viperfx/code/redwood-app/node_modules/.bin/rw run push
rw run

Run your script

Positionals:
  name  The name of the script to run                            [string]

Options:
  --help     Show help                                          [boolean]
  --version  Show version number                                [boolean]

Also see the Redwood CLI Reference

Unknown argument: push
```

So currently only `yarn rw run --name <name>` works